src/ethereum_test_specs/helpers.py

```diff
@@ -1,71 +1,93 @@
 """Helper functions."""
 
 from dataclasses import dataclass
-from typing import Any, Dict, List
+from typing import Any, Dict, List, Literal
 
 import pytest
 
 from ethereum_clis import Result
-from ethereum_test_exceptions import ExceptionBase, ExceptionMapper, UndefinedException
+from ethereum_test_exceptions import (
+    BlockException,
+    ExceptionBase,
+    ExceptionWithMessage,
+    TransactionException,
+    UndefinedException,
+)
 from ethereum_test_types import Transaction, TransactionReceipt
 
 
-class TransactionUnexpectedSuccessError(Exception):
+class UnexpectedSuccessError(Exception):
```

Comment:
I understand the exception rename, but I think the new name is a little too generic. Made a suggestion in #1404.